### DISCUSSION TITLE
[gatsby-starter-minimal-blog] react components and style

### DISCUSSION THREAD

**Discussion by giunab** [LekoArts/gatsby-themes#970] [Questions]
Hi,

is there a possibility to Integrator react components into a new created page and use the theme styling?

thanks

**Comment by LekoArts** [comment]
Sure, you can use Theme UI to style your page (see https://github.com/LekoArts/gatsby-themes/tree/main/themes/gatsby-theme-minimal-blog/src) on how it's done and then the underlying Theme UI theme is used.
